src/frontend/efiling-frontend/src/components/page/package-confirmation/PackageConfirmation.js

```diff
@@ -25,45 +26,105 @@ const uploadButton = {
   styling: "normal-white btn"
 };
 
-// TODO: Fix values
-const feesData = [
-  {
-    name: "Statutory Fees:",
-    value: "$100.00",
-    isValueBold: true
-  },
-  {
-    name: "Number of Documents in Package:",
-    value: "10",
-    isValueBold: true
-  }
-];
+const calculateTotalFee = files => {
+  let totalStatFee = 0;
+  files.forEach(file => {
+    totalStatFee += file.statutoryFeeAmount;
```

Comment:
Should you check for undefined or null ?

Reply:
you mentioned you will always return a number (such as 0), so I was not doing any checks here

Reply:
is this not the case anymore?

Reply:
Yes it is the case, so we should be good...